.github/cmake/aarch64-linux-gnu.cmake

```diff
@@ -0,0 +1,7 @@
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR ARM64)
+set(CMAKE_C_COMPILER aarch64-linux-gnu-gcc)
+set(CMAKE_CXX_COMPILER aarch64-linux-gnu-g++)
```

Comment:
CMake cannot find proper environment for linux target?

Reply:
I am not usre I understand the question? By default CMake detects compiler of the host system. If one is doing cross-compilation he/she should use such toolchain file.

Reply:
Is there a way to specify only the prefix?
aarch64-linux-gnu

Reply:
None that I am aware of in a sense that only one CMake variable would be set in the toolchain file. Actually, this is a better documentation on the topic https://cmake.org/cmake/help/book/mastering-cmake/chapter/Cross%20Compiling%20With%20CMake.html.

We can declare and re-use our own variable, if you prefer this way?
```
set(GCC_TARGET_PREFIX aarch64-linux-gnu-)
set(CMAKE_SYSTEM_NAME Linux)
set(CMAKE_SYSTEM_PROCESSOR ARM64)
set(CMAKE_C_COMPILER "${GCC_TARGET_PREFIX}gcc")
set(CMAKE_CXX_COMPILER "${GCC_TARGET_PREFIX}g++")
set(CMAKE_RC_COMPILER "${GCC_TARGET_PREFIX}windres")
set(CMAKE_RANLIB "${GCC_TARGET_PREFIX}ranlib")
set(CMAKE_AR "${GCC_TARGET_PREFIX}ar")
```

Reply:
I see, up to you then.